blocks/theme-editor/theme-editor.js

```diff
@@ -86,15 +86,23 @@ export default async function decorate(block) {
       </div>`;
 
     // Load site theme
-    fetch(`https://preview--${id}.${KESTREL_ONE}/styles/vars.css`)
+    const cssVarsData = await fetch(`https://preview--${id}.${KESTREL_ONE}/styles/vars.css`)
```

Comment:
This file is hard to review since you unindented mostly the whole file now all lines appear as changed in the PR.

Reply:
https://github.com/headwire-edge-delivery/eds-self-service-website/pull/25/commits/c3f33770879d35ec8101f8efc44296e3dd14cecc

Separate commit where I ported the changes